lib/proxy/ip-utils.php

```diff
@@ -16,121 +16,117 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class IpUtils
```

Comment:
We should probably exclude this lib and the related test file (`tests/test-lib-proxy-ip-utils.php`) since it's all taken from a 3rd party lib.